src/main/java/teammates/ui/controller/StudentFeedbackResultsPageData.java

```diff
@@ -137,7 +137,9 @@ private FeedbackResultsQuestionDetails createQuestionDetails(
                     filterResponsesByRecipientEmail(recipient, responsesBundle);
             
             boolean isUserRecipient = student.email.equals(recipient);
-            boolean isUserTeamRecipient = question.recipientType == FeedbackParticipantType.TEAMS
+            boolean isUserTeamRecipient = (question.recipientType == FeedbackParticipantType.TEAMS
+                                          || question.recipientType == FeedbackParticipantType.CUSTOM
+                                                  && question.isFeedbackPathsRecipientTypeTeams())
```

Comment:
Likewise, we could make a method `isRecipientTeam` in FeedbackQuestionAttributes